.github/workflows/create_release.yml

```diff
@@ -0,0 +1,51 @@
+name: Create Release
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        required: true
+        type: string
+
+jobs:
+  release:
+    permissions:
+      # Necessary permissions to create a release.
+      contents: write
+    env:
+      BUILD_DIR: build
+      RELEASE_BIN_ARCHIVE: release_bin.tar.gz
+      RELEASE_SRC_ARCHIVE: release_src.tar.gz
+      DEFAULT_BRANCH: refs/heads/${{ github.event.repository.default_branch }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      # Update the package index, then install all dependencies listed in
+      # the various apt-requirements.txt files in the project.
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ninja-build libpixman-1-dev libgtk-3-dev
```

Comment:
I do not think GTK is required for Opentitan project. It should be only useful for `ibexdemo` - where the SPI screen is emulated to display the Mandelbrot fractal :-). If you do not plan to test the `ibexdemo` machine, GTK dependency (and `--enable-gtk`) can be skipped to speed up build.

Reply:
Yes that's true that we don't need it for ibex, I'll remove that.